### PR TITLE
[#1350] Allow text formatting for text area inputs in group pages

### PR DESCRIPTION
**Pull request by mauriel133**
Also contains minor layout fixes for default 'no items found' labels.

### CODE REVIEW

**Comment by pkdash** [comment]
@shawncrawley Can you review this?

File: theme/templates/pages/group.html [hunk 1]
```diff
@@ -467,7 +467,7 @@ <h4 class="panel-title">
                                     <div class="panel-body">
                                         <div class="list-group">
                                             <ul class="list-group inputs-group">
-                                                <h5><i>No resources have been shared with this group yet.</i></h5>
+                                                <i class="list-group-item no-items-found"><i>No resources have been shared with this group yet.</i></i>
```
Comment (shawncrawley) [inline theme/templates/pages/group.html]: Why are there two <i> tags here, one nested inside the other?

Comment (mauriel133) [inline theme/templates/pages/group.html]: This is a mistake. I'll have it fixed.

Comment (mauriel133) [inline theme/templates/pages/group.html]: I've made the change now.

**Comment by shawncrawley** [comment]
+1

**Pull request merged**